client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -148,7 +148,7 @@ export class PluginsBrowser extends Component {
 					context: 'Category description for the plugin browser.',
 				} );
 			case 'featured':
-				return translate( 'Featured', {
+				return translate( 'Staff Picks', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Staff Picks', { context: 'Theme Showcase term name'} )` *ES Score: 19*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plugin-cards)


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?